platform/ios/src/MGLCameraChange.h

```diff
@@ -0,0 +1,35 @@
+#import "MGLFoundation.h"
+
+typedef NS_OPTIONS(NSUInteger, MGLCameraChangeReason)
+{
+    /// No reason for the camera move is specified.
+    MGLCameraChangeReasonNone = 0,
+
+    /// Set when a public API that moves the camera is called. This may be set for some gestures
```

Comment:
Text in documentation comments should always be styled as complete sentences (even when they’re sentence fragments).